selenium-system-test/src/test/scala/hts/pages/AuthorityWizardPage.scala

```diff
@@ -98,7 +98,7 @@ object AuthorityWizardPage extends Page {
   def setNino(nino: String)(implicit driver: WebDriver): Unit =
     Browser.find(Browser.name("nino")).foreach { element ⇒
       element.underlying.sendKeys(nino)
-      println("NINO entered on auth wizard: " + nino)
+      ScenarioContext.currentNINO
```

Comment:
this line isn't doing anything - the function returns `Unit` and this line returns a `String`